src/pudl/etl/__init__.py

```diff
@@ -58,7 +57,9 @@
                 "resources": {
                     "dataset_settings": {
                         "config": EtlSettings.from_yaml(
-                            importlib.resources.path(settings, "etl_fast.yml")
+                            importlib.resources.path(
+                                "pudl.package_data.settings", "etl_fast.yml"
+                            )
```

Comment:
I ended up just unignoring `pudl.package_data` in conf.py in #2306. Seems like using the string is a better option.

Reply:
Yeah I was dealing with the same problem, and this seemed easiest. I don't have a strong preference between the solutions, but I wasn't sure if un-ignoring the module would break something else

Reply:
Un-ignoring the module would just create docs for the package_data subpackage which I don't think we need. I think this is the correct solution.